recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/xen/xen_git.bbappend

```diff
@@ -30,4 +30,6 @@ do_deploy_append_rcar () {
     if [ -f ${D}/boot/${FLASK_POLICY_FILE} ]; then
         ln -sfr ${DEPLOYDIR}/${FLASK_POLICY_FILE} ${DEPLOYDIR}/xenpolicy
     fi
+
+    find ${S}/xen -iname "xen-syms" -exec tar -cJvf ${DEPLOYDIR}/xen-syms.tar.xz {} \;
```

Comment:
xen-syms location is known. It is located at `${S}/xen/xen-syms`. No need to use `find` there.

Reply:
it was just to ensure it is really there, let's assume it is...